modules/s2i/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -14,3 +14,24 @@ source "${JBOSS_CONTAINER_JAVA_S2I_MODULE}/maven-s2i-overrides"
 # invoke the build
 maven_s2i_build
 
+# run the pathfinder scripts to define JAVA_APP_JAR and JAVA_LIB_DIR
```

Comment:
we don't need these changes (yet), please let's leave them for the jlink-dev PR.